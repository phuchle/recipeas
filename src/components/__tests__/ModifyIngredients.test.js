import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import ModifyIngredients from '../ModifyIngredients';

const props = {
  nextButton: true,
  plusButton: true,
  ingredientList: []
};

describe('ModifyIngredients', () => {
  test('renders without crashing', () => {
    const component = shallow(<ModifyIngredients {...props} />);
    const tree = toJson(component);

    expect(tree).toMatchSnapshot();
  });
});
