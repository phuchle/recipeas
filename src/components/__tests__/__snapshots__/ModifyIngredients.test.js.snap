// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ModifyIngredients renders without crashing 1`] = `
<div>
  <h4>
    Ingredients
    <Link
      replace={false}
      to="/search-ingredient"
    >
      <PlusButton
        className="pull-right"
      />
    </Link>
  </h4>
  <ListGroup
    bsClass="list-group"
    style={
      Object {
        "marginTop": "25px",
      }
    }
  />
  <Pager
    bsClass="pager"
  >
    <Link
      replace={false}
      to="/review"
    >
      <Button
        active={false}
        block={true}
        bsClass="btn"
        bsSize="large"
        bsStyle="primary"
        disabled={false}
      >
        Next
      </Button>
    </Link>
    <Link
      replace={false}
      to="/modify-title"
    >
      <Button
        active={false}
        block={true}
        bsClass="btn"
        bsSize="large"
        bsStyle="default"
        disabled={false}
      >
        Back
      </Button>
    </Link>
  </Pager>
</div>
`;
